easybuild/easyconfigs/k/kneaddata/kneaddata-0.10.0-GCC-10.3.0.eb

```
@@ -0,0 +1,38 @@
+#  kneaddata
+easyblock = 'PythonPackage'
+
+name = 'kneaddata'
+version = '0.10.0'
+
+homepage = 'https://huttenhower.sph.harvard.edu/kneaddata/'
+description = """
+ KneadData is a tool designed to perform quality control on metagenomic sequencing data, especially data from
+ microbiome experiments. In these experiments, samples are typically taken from a host in hopes of learning something
+ about the microbial community on the host. However, metagenomic sequencing data from such experiments will
+ often contain a high ratio of host to bacterial reads. This tool aims to perform principled in silico separation
+ of bacterial reads from these “contaminant” reads, be they from the host, from bacterial 16S sequences,
+ or other user-defined sources.
+"""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+#  source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['ae43a1ebafbf9bc282492178eb632a6b2e8d1e4c57d691419e19050f20488425']
+
+
+download_dep_fail = 'true'
+dependencies = [
+    ('Java', '1.8', '', SYSTEM),
+    ('Trimmomatic', '0.38-Java-1.8', '', SYSTEM),
+    ('FastQC', '0.11.5-Java-1.8.0_74', '', SYSTEM),
```

default Java version used in this toolchain generation is Java 11, so we should use those versions if possible:

```suggestion
    ('Java', '11', '', SYSTEM),
    ('Trimmomatic', '0.39', '-Java-%(javaver)s', SYSTEM),
    ('FastQC', '0.11.9', '-Java-%(javaver)s', SYSTEM),
```